src/js/libgeo/GeoRender.js

```diff
@@ -241,6 +241,30 @@ function drawgeotext(el) {
         evaluator.drawtext$2([pos, text], opts, htmlCallback);
 }
 
+function drawgeopolygon(el) {
+    if (!el.isshowing || el.visible === false)
+        return;
+
+    var modifs = {};
+    modifs.color = el.color;
+    modifs.alpha = el.alpha;
+    modifs.size = el.size;
```

Comment:
Shouldn't these properties be set by `polygonDefault`? Otherwise they won't be wrapped as CindyJS data types.